scss/_settings_spacing.scss

```diff
@@ -75,8 +75,9 @@ $spv--small: $sp-unit !default;
 $spv--medium: $sp-unit * 1.5 !default;
 $spv--large: $sp-unit * 2;
 $spv--x-large: $sp-unit * 3 !default;
-$spv--strip-regular: $sp-unit * 8 !default;
-$spv--strip-deep: $sp-unit * 12 !default;
+$spv--strip-shallow: $sp-unit * 3 !default;
+$spv--strip-regular: $sp-unit * 8 !default; // will be changed to 6
```

Comment:
I don't think this one is meant to be changed. Regular spacing should remain at 4rem, so value of 8 spacing units is correct.